lib/cisco_node_utils/command_reference.rb

```diff
@@ -260,6 +260,32 @@ def initialize(product:  nil,
       if files
         @files = files
       else
+        @files = []
```

Comment:
This is still based on old develop code.  We have changed the way yaml files are processed.

Please sync your fork with latest develop.  Here is the process:

1) https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced
2) https://help.github.com/articles/syncing-a-fork/

Once the sync is complete the yaml will need to be refactored.  Here is the complete README.md describing the new yaml architecture and features.

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/README_YAML.md